src/lib/puppeteer/Puppeteer.js

```diff
@@ -24,6 +24,25 @@ class Puppeteer {
     await page.goto( `http://localhost:${config.api.port}/docs/${filename}.html`, { waitUntil: 'networkidle2' });
     await page.pdf({ path: `tmp/${filename}.pdf`, ...combinedOptions });
   }
+
+  async renderURLToPDF ( filename, pdfOptions={}, otherOptions={}){
```

Comment:
Nice work on this function! Ideally I'd like to combine the options field into a single object. Maybe a better method prototype would be `renderURLToPDF( filename, data, options )`. Thoughts?